app/assets/javascripts/application.js

```diff
@@ -1,3 +1,11 @@
+//= require govuk_publishing_components/lib
```

Comment:
Can we have these after L9? (`//= require govuk-frontend/govuk/vendor/polyfills/Function/prototype/bind`) to preserve the dependency tree.

Reply:
Updated. Apols, I was following the advice in the component guide `Add this to your application's main js file, before your other local imports.` - does this need clarifying?

Reply:
Rightly so. I didn't anticipate having applications that would require polyfills from `govuk-frontend` – will update the guidance accordingly.